build.js

```diff
@@ -60,14 +68,47 @@ StyleDictionaryPackage.registerFilter({
   name: 'excludeBrand',
   matcher: function (token) {
     return token.category !== 'brand';
-  }
+  },
 });
 
+/**
+ * REGISTER FORMATS
+ * @see https://amzn.github.io/style-dictionary/#/api?id=registerformat
+ */
+
 StyleDictionaryPackage.registerFormat({
   name: 'json/flat',
   formatter: function (dictionary) {
     return JSON.stringify(dictionary.allProperties, null, 2);
-  }
+  },
+});
+
+StyleDictionaryPackage.registerFormat({
+  name: 'javascript/nested',
+  formatter: function (dictionary) {
+    const tokens = dictionary.properties;
+    const removeMetadataAndFlatten = (jsonObj) => {
+      for (var key in jsonObj) {
+        if (typeof jsonObj[key] == 'object') {
+          if (jsonObj[key].hasOwnProperty('value')) {
+            jsonObj[key] = jsonObj[key]['value'];
+          } else {
+            removeMetadataAndFlatten(jsonObj[key]);
+          }
+        } else {
+          continue;
+        }
+      }
+      return jsonObj;
+    };
+    const flattenedTokens = removeMetadataAndFlatten(tokens);
```

Comment:
the formatter provides a lot of metadata out of the box along with the token's value, which I believe we don't really need. This script will flatten them up and assign the `value` key's value to the token name. ex:

```JSON
"neutrals": {
      "grey07": {
        "value": "#555555",
        "type": "colors",
        "filePath": "tokens/globals/colors/colors.json",
        "isSource": true,
        "original": {
          "value": "{alias.colors.brand.neutrals.grey07}",
          "type": "colors"
        },
        "name": "TOKEN_COLOR_NEUTRALS_GREY07",
        "attributes": {},
        "path": [
          "colors",
          "neutrals",
          "grey07"
        ]
      },
}
```
becomes 
```JSON
"neutrals": {
      "grey07": "#555555",
}
```

Reply:
I will comment and add tests if we're sticking with this